plugins/woocommerce/includes/class-wc-privacy.php

```diff
@@ -249,6 +250,33 @@ public static function trash_cancelled_orders( $limit = 20 ) {
 		);
 	}
 
+	/**
+	 * Find and Anonymize refunded orders.
+	 *
+	 * @since 9.7.0
+	 * @param  int $limit Limit orders to process per batch.
+	 * @return int Number of orders processed.
+	 */
+	public static function anonymize_refunded_orders( $limit = 20 ) {
+		$option = wc_parse_relative_date_option( get_option( 'woocommerce_anonymize_refunded_orders' ) );
+
+		if ( empty( $option['number'] ) ) {
+			return 0;
+		}
+
+		return self::trash_orders_query(
```

Comment:
@masteradhoc If I understood correctly we want to anonymize, not trash, right? Should this be `anonymize_orders_query`?

Reply:
@albarin yes totally correct, fixed it!